calculateXSec.py

```diff
@@ -0,0 +1,401 @@
+import argparse	
```

Comment:
Why is is this file called calculateXSec.py?

Reply:
Sorry I forgot to delete this file........ (Just for test purpose)